modules/apps/foundation/frontend-theme/justin-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,461 @@
+/* These inject tags are used for dynamically creating imports for themelet styles, you can place them where ever you like in this file. */
+
+/* inject:imports */
+/* endinject */
+
+/* This file allows you to override default styles in one central location for easier upgrade and maintenance. */
+
+@import url('https://fonts.googleapis.com/css?family=Merriweather:400,400i,700,700i');
+@import url('https://fonts.googleapis.com/css?family=Montserrat:400,500,500i,600,600i,700,700i,800,800i');
+
+$p-size: 12pt;
+$grey: #d1d1d1;
+$blue: #ff8600; //actually orange
+//$blue: #007acc;
+
+%border-props{
+	border-style: solid;
+	border-color: $grey;
+	border-width: 1px;
+	}
+
+%padding-highlight{
+	padding-top: .125em;
+	padding-bottom: .125em;
+	padding-right: .25em;
+	padding-left: .25em;
+	}
+
+%form-focus{
+	background-color: #FFF;
+	border-color: $blue;
+	box-shadow: 0 0 5px $blue;
+	outline: none;
+}
+
+html {
+
+	height: 100%;
+//////////////////////////////////////////
+/////////////WRAPPER/////////////////////
+////////////////////////////////////////
+#wrapper{
+	height: 100%;
+	font-family: merriweather, georgia;
+	background-color: rgb(32, 32, 32);
+	line-height: 1.75;
+	font-size: $p-size;
+
+	::selection{
+		color: #fff;
+		background: $blue;
+	}
+	::-moz-selection{
+		color: #fff;
+		backgorund: $blue;
+	}
+
+	&::before{
+		top: 0; //64 is the height of the control menu, try to see if there is a dynamic way of doing this.
+		height: 96px;
+		}
+
+	&::before, &::after{
+		background: inherit;
+		content: "";
+		display: block;
+		left: 0;
+		position: fixed;
+		width: 100%;
+		z-index: 99;
+
+		}
+
+	&::after{
+		bottom: 0;
+		height: 24px;
+		}
+
+	#wrapper2{
+		background-color: rgb(255, 255, 255);
+		padding: 20px;
+		}
+
+	a{
+		text-decoration: underline;
+		color: $blue;
+		&:hover{
+			text-decoration: none;
+			color: $grey;
+			}
+		}
+
+
+	li{
+		a{
+			color: $blue;
+			text-decoration: none;
+			}
+		}
+
+	// more efficient way of selecting this by type?
+	input[type=password], [type=text], [type=url], [type=email], [type=tel], [type=search], [type=number], [type=color], [type=number], [type=range], [type=date], [type=month], [type=week], [type=datetime], [type=datetime-local], textarea{
+		@extend %border-props;
+		border-radius: 1px;
+		//font-size: $p-size - 2pt;
+		background-color: #F5F5F5;
+		//opacity: .6;
+		padding-left: .5em;
+
+		&::placeholder, &[value]{
+			font-family: montserrat;
+			font-size: $p-size - 2pt;
+			}
+
+		&:focus{
+			@extend %form-focus;
+			}
+		}
+
+	//input buttons
+	input[type=submit],[type=button],[type=reset], button{
+		@extend %border-props;
+		border-radius: 3px;
+		background-color: #fff;
+		color: $blue;
+		&:hover{
+			background-color: #f5f5f5;
+			}
+		}
+
+	select{
+		background-color: #f5f5f5;
+		&:focus{
+			@extend %form-focus;
+		}
+	}
+
+
+
+		// a{
+		// 	&:hover{
+		// 		color: $blue;
+		// 	}
+
+		// }
+}
+
+@media (max-width: 767px){
```

Comment:
Changed to Liferay default value `$screen-xs-max` instead of `767px`.

https://github.com/liferay/liferay-portal/blob/master/modules/apps/foundation/frontend-css/frontend-css-common/src/main/resources/META-INF/resources/liferay/_variables.scss